hc-nested-list-behavior.html

```diff
@@ -107,7 +122,7 @@
      */
     itemOpenChanged(event, detail, idName = this.itemIdName, parts = this.parts) {
       // Block interactions while loading items
-      if (this.isLoading) return;
+      if (this.isLoading || !parts) return;
```

Comment:
I thought that would be the case as well but I was noticing that fn throwing an error in mtz-group-manager because parts was undefined.